pkg/scripts/uninstall.sh

```diff
@@ -87,7 +88,7 @@ destroy_infrastructure() {
   cd "${SCRIPT_PATH}/../../"
   set +e
   # Start destroying the infrastructure
-  terraform destroy "${OVERRIDE_CONFIG_FILE}"
+  terraform destroy "${OVERRIDE_CONFIG_FILE}" | tee "${LOG_FILE}"
```

Comment:
The same as https://github.com/atlassian-labs/data-center-terraform/pull/65/files#r762721199